docs/docs-source/docs/modules/cli/pages/cloudflow_deploy.adoc

```diff
@@ -59,25 +92,23 @@ the stored credentials.
 
 You can update the credentials with the "update-docker-credentials" command.
 
-[source,bash]
-----
-kubectl cloudflow deploy [flags]
-----
 
 == Examples
 
 [source,bash]
 ----
-kubectl cloudflow deploy registry.test-cluster.io/cloudflow/sensor-data-scala:292-c183d80 valid-logger.log-level=info valid-logger.msg-prefix=valid
+$ kubectl cloudflow deploy call-record-aggregator.json valid-logger.log-level=info valid-logger.msg-prefix=valid
```

Comment:
Looks like an incorrect example? (deploy, but also it shows the old way for specifying config parameters)

Reply:
(Rather remove the valid-logger.. settings her, I'll check the docs in the code)